site/docs/try/authentication.md

```diff
@@ -0,0 +1,25 @@
+# Authentication
+
+By default, Nessie servers run with authentication disabled and all requests are processed under the "anonymous"
```

Comment:
Probably worth a sentence that there are other pages about configuring authn for the Nessie clients and link to those.

Reply:
Fixed in 3b95a50929b5664beab05ed75e0380a42d703a6a